app/views/layouts/_footer.html.erb

```diff
@@ -101,6 +101,9 @@
       &copy; 2019<%= "-#{Date.today.year}" if Date.today.year > 2019 %>
       <%= FMA_LLC.capitalize %> <%= t('footer.copyright_text') %>
       (<%= request.env['SERVER_NAME'] %>)
```

Comment:
Please remove server name from this place, since it's now at the end of the line.